docs/customresourcestate-metrics.md

```diff
@@ -49,7 +49,7 @@ spec:
           - --resources=certificatesigningrequests,configmaps,cronjobs,daemonsets,deployments,endpoints,foos,horizontalpodautoscalers,ingresses,jobs,limitranges,mutatingwebhookconfigurations,namespaces,networkpolicies,nodes,persistentvolumeclaims,persistentvolumes,poddisruptionbudgets,pods,replicasets,replicationcontrollers,resourcequotas,secrets,services,statefulsets,storageclasses,validatingwebhookconfigurations,volumeattachments,verticalpodautoscalers
 ```
 
-NOTE: The `group`, `version`, and `kind` common labels are reserved, and will be overwritten by the values from the `groupVersionKind` field.
+NOTE: The `cr_group`, `cr_version`, and `cr_kind` common labels are reserved, and will be overwritten by the values from the `groupVersionKind` field.
```

Comment:
I would prefer if we had some consistency between this prefix and the one in the metric name. Currently the metric name is using the `crd` prefix which might not be well suited. Might be better to go with `kube_cr` or `kube_custom_resources`.

Reply:
do you mean we should use the future prefix in the metric name and in the label itself?
I like `kube_custom_resource` more than `kube_cr`.

Reply:
Yes, I'd like to avoid having `crd` somewhere and `cr` elsewhere, it's better be consistent.

Reply:
I agree, would be nice to have one thing. I believe right now we mix Custom Resource, Custom Resource Definition and Custom Resource State.  `kube_customresource` would align with `kube_horizontalpodautoscaler` (no additional underscores).

Should we rename everything custom resource state into simply custom resource and abbreviate it with "cr" where necessary?

Reply:
i've pushed a new commit (https://github.com/kubernetes/kube-state-metrics/pull/1942/commits/ed85920888a27a984f5f1a65c6e2d7caedee2948) - this should reflect the way we want to go further:

Reply:
FYI: Based on the previous commit i started to improve the e2e tests because they didn't cover the default generated `group`,`version`,`kind` labels atm. will continue on monday morning.